core/dcschema.mcf

```diff
@@ -2858,3 +2858,24 @@ Node: dcid:Portion
 typeOf: dcs:UnitOfMeasure
 name: "Portion"
 description: "A unit of measure used to measure serving of food."
+
+Node: dcid:exchangeRate
+typeOf: schema:Property
+name: "exchangeRate"
+description: "The price of a base currency need in order to exchange to a single unit of another currency, e.g. if it exchange rate of EUR to GBP is 1.2, it takes 1.2 EUR to exchange for 1 GBP."
```

Comment:
"base currency need in order to exchange to "  do you mean "needed"?

FYI: a convenience thing we do is copy/paste from wikipedia and link to it as 
descriptionUrl: "https://en.wikipedia.org/wiki/Exchange_rate"

Reply:
I think Prashanth meant that we often quote an external source for the description, at which point we would note the source using descriptionUrl. If we keep the description here, then we should not add descriptionUrl

Reply:
Gotcha. Reading wiki, their definition sounds more general,like what the description here originally was. Given that our current description is more detailed and includes base price reference with an example, I'd vote to keep ours. Thoughts?